Mu2eUtilities/inc/BinnedSpectrum.hh

```diff
@@ -50,6 +52,7 @@ namespace mu2e {
     double         getXMaxUnbinned() const { return _xmax_unbinned;}
     double         getXMax() const { return _xmax;}
     double         getXMin() const { return _xmin;}
+    std::string    spectrumVariable() const { return _spectrumVariable;}
```

Comment:
This is still adding extra stuff to BinnedSpectrum.   Which BinnedSpectrum has no use for.  Please put spectrumVariable  configuration instead into tools that need it, like EventGenerator/src/MuCapNeutronGenerator_tool.cc   I think you do not need to modify BinnedSpectrum from its original version.   The fhicl config will look like

   spectrumVariable: kineticEnergy
   spectrum: { ... }

instead of having spectrumVariable inside; I think this is fine.

Reply:
Thanks, I was having issues getting spectrumVariable out of spectrum so put it into BinnedSpectrum as that seemed like the simplest solution. I will move it into the tool config like you suggest.

Reply:
Done in commit 429a1beba

Reply:
> Done in commit [429a1be](https://github.com/Mu2e/Offline/commit/429a1beba1ff881af84ed52bbe659df05bd0928c)

Thank you!